test/remoting.integration.js

```diff
@@ -235,6 +235,32 @@ describe('With model.settings.replaceOnPUT true', function() {
   });
 });
 
+describe('Model shared classes : upsertWithWhere', function() {
+  it('has expected remote methods', function() {
+    var storeClass = findClass('store');
+    var methods = getFormattedMethodsExcludingRelations(storeClass.methods);
+    var expectedMethods = [
+      'create(data:object):store POST /stores',
+      'patchOrCreate(data:object):store PATCH /stores',
+      'replaceOrCreate(data:object):store PUT /stores',
+      'replaceOrCreate(data:object):store POST /stores/replaceOrCreate',
+      'exists(id:any):boolean GET /stores/:id/exists',
+      'findById(id:any,filter:object):store GET /stores/:id',
+      'replaceById(id:any,data:object):store PUT /stores/:id',
+      'replaceById(id:any,data:object):store POST /stores/:id/replace',
+      'find(filter:object):store GET /stores',
+      'findOne(filter:object):store GET /stores/findOne',
+      'upsertWithWhere(where:object,data:object):store POST /stores/upsertWithWhere',
+      'updateAll(where:object,data:object):object POST /stores/update',
+      'deleteById(id:any):object DELETE /stores/:id',
+      'count(where:object):number GET /stores/count',
+      'prototype.patchAttributes(data:object):store PATCH /stores/:id',
+      'createChangeStream(options:object):ReadableStream POST /stores/change-stream',
```

Comment:
Please move this test to `describe('Model shared classes', ` (the block which starts on line 76) and assert only on the methods related to your patch.

```js
it('has upsertWithWhere remote method', function() {
  var storeClass = findClass('store');
  var methods = getFormattedMethodsExcludingRelations(storeClass.methods);
  var expectedMethods = [
    'upsertWithWhere(where:object,data:object):store POST /stores/upsertWithWhere',
  ];
  expect(methods).to.include.members(expectedMethods);
});
```